lib/packer/compiler.rb

```diff
@@ -59,7 +59,7 @@ def run_webpack
 
       logger.info "ENV: #{webpack_env}"
 
-      sterr, stdout, status = Open3.capture3(webpack_env, 'npx webpack')
+      sterr, stdout, status = Open3.capture3(webpack_env, 'yarn run webpack')
```

Comment:
`run` is optional, but i suppose good here those that might not be familiar with what `yarn` is doing.